_posts/2024-01-06-developer-mission-statement.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+id: 4c439aca-9f8f-4e66-9589-293f0481ab1a
+title: My Mission Statement as a Programmer
+author: Mick F
+excerpt: >-
+  Climate Friendliness, Human Friendliness, Exigence, IndieWeb Friendliness, Big
+  Corporation Cautiousness, Durability & Sustainability serves as a guiding
+  compass in my journey as a programmer
+category: Column
+tags:
+  - iOS
+  - Swift
+---
+
+## tl;dr
+
+As a programmer, I create eco-friendly software with user-friendly design,
+embracing IndieWeb principles. I'm cautious with big corporations, upholding
+ethics and user rights. My goal is durable, sustainable code, prioritizing
+maintainability and developer well-being. I respond swiftly to evolving
+challenges for impactful work in a changing tech landscape.
+
+## A longer version
+
+Let me explain. There is a sense of uneasiness with the state of the world these
+days. And after I had the privilege to spend a fun and peaceful holiday season,
+I also wanted to give a headcheck on where I stand with my role as a programmer
+and if I am aligned with my aspirations and values. So I decided to write my
+mission statement. Let me develop each of its components, with an example of how
+I think I apply it.
+
+### 🌍 Climate Friendliness
+
+I need to consider all environmental impacts of the code I write: the hardware
+it runs on, the energy it requires, its purpose for the people running it.
+
+Example. I work for [Memo Bank][18], a bank for companies, that [acts][1]
+towards the transition to a low carbon economy. It has been recognised as such
```

Comment:
```suggestion
the transition to a low carbon economy. It has been recognised as such
```

Reply:
```suggestion
towards the transition to a low carbon economy. It has been recognised as for its approach
```

Reply:
```suggestion
towards the transition to a low carbon economy. It has been recognised for its progressive approach
```

Reply:
Maybe add a little conclusion on this point? something like : "This is one of the main values I really appreciate about working at Memo Bank. Not only does it make me proud to work here, but it helps me learn and stay informed about this important issue, as Memo is continuously providing training and other resources on the subject for us employees.